Gopkg.toml

```diff
@@ -198,7 +198,15 @@ required = [
 
 [[constraint]]
   name = "google.golang.org/grpc"
-  version = "1.5.2"
+  version = "1.13.0"
+
+[[override]]
+  name = "golang.org/x/sys"
+  revision = "1c9583448a9c3aa0f9a6a5241bf73c0bd8aafded"
```

Comment:
could you explain why you need this override?

Reply:
Needed to upgrade the version as it was conflicting with the transitive protobuf dependency for auto-generated code bits (was complaining of some missing methods)